control/autoware_autonomous_emergency_braking/src/node.cpp

```diff
@@ -416,6 +418,14 @@ void AEB::onCheckCollision(DiagnosticStatusWrapper & stat)
       }
     }
     addVirtualStopWallMarker(virtual_wall_marker);
+
```

Comment:
I think it is necessary to output a metric when the decision is "none" too? 
The simplest way to do it would be in the onCheckCollision method I believe.